index.bs

```diff
@@ -2655,17 +2655,17 @@ Council Deliberations</h5>
 	<p id=broker-consensus>
 	The Council <em class=rfc2119>may</em>
 	further attempt to broker consensus,
-	which, if successful, disposes the formal objection.
+	which, if successful, disposes that formal objection.
```

Comment:
Since more than one FO can be handled by a Council in relation to a single decision:

```suggestion
	further attempt to broker consensus on one or more of the formal objections
	which, if successful, would allow them to be resolved.
```